Add comparison to other ad networks (#267)
readthedocs/ethicalads.io: content/pages/vs-carbon-ads.md

> Title: What Makes EthicalAds a Great Carbon Ads Alternative

Reviewer: ```suggestion
Title: What Makes EthicalAds a Great Carbon Ads Alternative for Your Site
```

I don't feel that strongly but this change might make it more obvious that this is geared toward publishers.

url: alternative-to-carbon-ads
save_as: alternative-to-carbon-ads/index.html
description: How are we different? Privacy guarantees, open source code, and community-focus.

We know that we are competing with a few other ad networks that focus on developers.
This page talks a bit more about how we're different,
and why you might want to choose us instead of Carbon Ads.

[TOC]

## **Privacy**: We never set a cookie on your site, or load third-party scripts.

When your user goes to your website, they are assuming they will only be sending data to you. Some users consider it a breach of trust if you were to share that data with other third parties, and it might be a breach of a privacy law to do this without user consent.

> These networks will often set cookies on your browser session, as well as load scripts hosted by Google into a publishers web page. You can see a screenshot of Carbon Ads loading a Google script here:

Reviewer: ```suggestion
These networks will often set cookies on your browser session, as well as load scripts hosted by Google into a publishers web page. You can see a screenshot of Carbon Ads loading a Google tracking pixel here:
```

Shots fired! Still, let's be precise.

<div class="m-5 postimage">
  <a href="{static}../images/posts/2021-third-party-resources.png">
  <img class="w-50 rounded mx-auto d-block"  src="{static}../images/posts/2021-third-party-resources.png" alt="An ad network loading third-party resources">

>   </a>

Reviewer: ```suggestion
  </a>
  <p><small>Carbon sometimes loads third-party resources from Google or other trackers</small></p>
```
